app/analyzers/creative_image_analyzer.rb

```diff
@@ -1,14 +1,20 @@
 class CreativeImageAnalyzer < ActiveStorage::Analyzer::ImageAnalyzer
   def metadata
-    md = super.dup
-    md[:name] = @blob.filename
-    return md unless md[:height].present? && md[:width].present?
+    super.dup.tap do |md|
+      md[:name] = @blob.filename
 
-    md[:format] = "icon" if md[:width].to_i == 20 && md[:height] == 20
-    md[:format] = "icon" if md[:width].to_i == 40 && md[:height] == 40
-    md[:format] = "small" if md[:width].to_i == 200 && md[:height] == 200
-    md[:format] = "large" if md[:width].to_i == 260 && md[:height] == 200
-    md[:format] = "wide" if md[:width].to_i == 512 && md[:height] == 320
-    md
+      if md[:height].present? && md[:width].present?
```

Comment:
You can just use `md.try(:[], :height).present?` for short.

* https://everydayrails.com/2011/04/28/rails-try-method.html

Reply:
That's a cool trick but I don't think it makes it any more readable.

Reply:
Right, it prolly was late at night when i reviewed this, for some reason I thought you were just checking for `height`, just noticed you are checking for `weight` presence as well.